diesel/src/connection/mod.rs

```diff
@@ -8,12 +8,19 @@ use types::HasSqlType;
 
 pub use self::transaction_manager::{TransactionManager, AnsiTransactionManager};
 
+/// To perform simple operations to the backed that implements this trait.
+/// `SimpleConnection` will be used for migrations. Every backend shall implement
+/// their own way to run migrations scripts.
```

Comment:
This is better, but still too many words for too little information. No need to make it longer than necessary – technical documentation is typically very terse. How about

> Perform simple operations on a backed.
>
> This is used for migrations.